x/concentrated-liquidity/pool.go

```diff
@@ -164,6 +164,17 @@ func (k Keeper) GetPoolDenoms(ctx sdk.Context, poolId uint64) ([]string, error)
 	return denoms, nil
 }
 
+// Return if the Pool has a position. This is guaranteed to be equi-satisfiable to
+// the current sqrt price being 0.
+// We also check that the current tick is 0, which is also guaranteed in this situation.
+// That derisks any edge-case where the sqrt price is 0 but the tick is not 0.
+func (k Keeper) PoolHasPosition(ctx sdk.Context, pool types.ConcentratedPoolExtension) bool {
```

Comment:
Would it be appropriate to define this method on the pool itself?

Reply:
Great idea!

Reply:
Actually on further thought I'm not sure, this is a convention set by the keeper not the pool itself